aviary/examples/reserve_missions/run_reserve_mission_fixedtime.py

```diff
@@ -40,8 +41,7 @@
 
 # Link phases and variables
 prob.link_phases()
-
-prob.add_driver("SLSQP", max_iter=100)
+prob.add_driver("SNOPT", max_iter=100, verbosity=Verbosity.VERBOSE)
```

Comment:
Does changing this to SNOPT negatively affect how this script is tested? Or does that not come into play?